firebase-firestore/src/main/java/com/google/firebase/firestore/util/CustomClassMapper.java

```diff
@@ -754,6 +757,7 @@ private Type resolveType(Type type, Map<TypeVariable<Class<T>>, Type> types) {
     }
 
     Map<String, Object> serialize(T object, ErrorPath path) {
+      // TODO: Add logic to skip @DocumentId annotated fields in serialization.
```

Comment:
Please don't check in naked "TODO" comments.  If this is intended to be checked in, please reference a bug ("TODO(b/xyz)") or your username for context.

Reply:
Done.